charts/rancher-monitoring/66.7.1-rancher.1/charts/prometheus-node-exporter/templates/daemonset.yaml.orig

```diff
@@ -49,7 +49,7 @@ spec:
       containers:
         {{- $servicePort := ternary .Values.kubeRBACProxy.port .Values.service.port .Values.kubeRBACProxy.enabled }}
         - name: node-exporter
-          image: {{ template "system_default_registry" . }}{{ include "prometheus-node-exporter.image" . }}
```

Comment:
Need to keep this for prime customers

Reply:
Oh great catch - if this is current state that was absolutely a mistake.

Reply:
Oh actually we don't need that - the image helper handles registry.

However I think while looking into this point, I've noticed some more images that need to be updated potentially.